examples/README.md

```diff
@@ -0,0 +1,55 @@
+# Manticore Examples
+
+## Quickstart
+
+Install and try Manticore in a few shell commands:
+
+```bash
+# Install system dependencies
+sudo apt-get update && sudo apt-get install python3 python3-dev python3-pip -y
```

Comment:
This feels a bit unnecessary outside of the top-level README. It should be pretty obvious that Python 3 is required now. If not, we could maybe point them to the Dockerfile if they want to know the exact dependencies?